sync/class.jetpack-sync-defaults.php

```diff
@@ -104,6 +104,8 @@ class Jetpack_Sync_Defaults {
 		'users_can_register',
 		'active_plugins',
 		'uninstall_plugins',
+		'advanced_seo_front_page_description',
+		'advanced_seo_title_formats',
 	);
```

Comment:
I like that, thanks! :+1: 

I've tried using consts just for the fun of it, but it turns out they are breaking sync and tests (not defined there, which makes sense). :)